mtags/src/main/scala-3/scala/meta/internal/pc/completions/Completions.scala

```diff
@@ -63,6 +64,48 @@ class Completions(
     case (_: Ident) :: (_: SeqLiteral) :: _ => false
     case _ => true
 
+  private def calculateTypeInstanceAndNewPositions =
+
+    path match
+      case head :: tail
+          if head.isInstanceOf[Select] || head.isInstanceOf[Ident] =>
+        shouldAddSquareBracket =
+          val span: Span = head.srcPos.span
+          if span.exists then
+            var i = span.end
+            while i < (text.length() - 1) && text(i).isWhitespace do i = i + 1
```

Comment:
This is related to this issue in Dotty https://github.com/lampepfl/dotty/issues/15750
because of which we had to implement this workaround to detect the existing type parameters to avoid inserting a duplicate extra bracket in Type Completion.

Reply:
can you add a comment with a link to the issue?